### PR TITLE
[qtfeedback] Improve vkb haptics

### PR DESCRIPTION
**Pull request by Thaodan**

### CODE REVIEW

File: sparse/usr/usr/lib/qt5/plugins/feedback/droid-vibrator-device.ini [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ress = 35
```
Comment (sledges) [inline sparse/usr/usr/lib/qt5/plugins/feedback/droid-vibrator-device.ini]: I believed you meant "Press"?

**Comment by Thaodan** [comment]
Yes, fixed.

**Comment by sledges** [comment]
The path is incorrect: /usr/usr/

**Comment by Thaodan** [comment]
Fixed that, saw that after your last review.

**Comment by sledges** [comment]
One last thing;
Every PR must appear in the .changes file when we build the configs (or any other package) and have a bug associated with.
Please create a MER bug and rename this commit to [...] ... Fixes MER#xyz

Thanks!

**Comment by sledges** [comment]
To start with good habits early on, the proper name has to have Fixes, otherwise it means the commit only contributes to the bug; "[qtfeedback] Improve vkb haptics. Fixes MER#1953"
Thanks for your patience

**Comment by Thaodan** [comment]
ok, done.

**Pull request merged**